dbt/profiles.yml

```diff
@@ -0,0 +1,17 @@
+pudl_dbt:
```

Comment:
It seems like by default dbt only uses 1 thread when running the tests, but using more definitely sped it up considerably for me. Is there some way can have it use whatever the number of available CPUs is? Or will this be getting run in a script where we can set it with a command line argument? How will this work once we get it into our asset checks?

Also wow, it ran all the tests locally in 12 seconds with 8 threads. Even referencing the nightly outputs it only took 2 minutes.